<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gerer_conv</class>
 <widget class="QDialog" name="gerer_conv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter convention </string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_nomEntreprise">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DATE FIN</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_conv">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="resaisir_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NOM ENTREPRISE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="retour">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>260</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_idConv">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>150</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>imprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resaisir">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>Resaisir</string>
     </property>
    </widget>
    <widget class="QLabel" name="message">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher les conventions</string>
    </attribute>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rechercher">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>280</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficherConv">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>291</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Liste des conventions</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trier_id">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier id</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>381</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche_id">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="trier_nom">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="retour_affiche">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>320</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>280</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
